### PR TITLE
correction in ducttype

### PR DESCRIPTION
**Pull request by AntonReiswich**
We added sections to the duct type and fixed a typo.

### CODE REVIEW

File: schema/cpacs_schema.xsd [hunk 1]
```diff
@@ -35975,6 +35975,7 @@ The fuel tank volume type should also be used for the wing fuel tank</xsd:docume
                     <xsd:element name="transformation" minOccurs="0" type="transformationType"/>
                     <xsd:element name="positionings" minOccurs="0" type="positioningsType"/>
                     <xsd:element name="segments" type="fuselageSegmentsType"/>
+                    <xsd:element name="sections" type="fuselageSectionsType"/>
```
Comment (MarAlder) [inline schema/cpacs_schema.xsd:35978]: Ups, I updated the official scheme accordingly https://github.com/DLR-SL/CPACS/commit/f5c31da64fea509d1c3eb8d7d10928e72f6fb6fa

Comment (joergbrech) [reply schema/cpacs_schema.xsd:35978]: Thanks @MarAlder. I suppose this PR can be closed then?

**Comment by luzpaz** [comment]
Is this PR still relevant ?

**Comment by MarAlder** [comment]
Thanks, all changes are implemented in the [DLR-SL:ducts](https://github.com/DLR-SL/CPACS/tree/ducts) branch. I'll therefore close this PR.

**Pull request closed without merging**